README.md

```diff
@@ -1 +1,132 @@
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Enumerable] <a name="about-project"></a>
+**[Enumerable]** In this project you will learn how to use a module inside your class. For this we created a class MyList and a module MyEnumerable. Our module MyEnumerable will implement a subset of the functionality of Enumerable
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a>Ruby</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Using Ruby language]**
+- **[Using Methods]**
+- **[Morse code]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- the Ruby should be installed in your machine.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- `git clone https://github.com/PedroH-Neves/ruby-enumerable/`
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Najibullah Jafari**
+
+- GitHub: [Najibullah_jafari](https://github.com/najibullahjafari)
+- Twitter: [Najibullah_jafari](https://twitter.com/Najib_Jafari_)
+- LinkedIn: [Najibullah_jafari](https://www.linkedin.com/in/najibullahjafari/)
+
+👤 **Pedro Henrique Domingues Neves**
+
+- GitHub: [@PedroH-Neves](https://github.com/PedroH-Neves)
+- Twitter: [@PedroHDN710](https://twitter.com/PedroHDN710)
+- LinkedIn: [Pedro Neves](https://www.linkedin.com/in/pedrohdn7)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Future Features
+
+- I think about how to make it fast.
+- I will make a class for it.
+- We will do our best to decrease duplication of code.
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/PedroH-Neves/ruby-enumerable/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project that would be nice to put a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for their helpful practices.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] @Team Great work on your readme, however, the MIT license link is not working, this is because you don't have a license in this branch, kindly consider adding a license file.

Reply:
- [ ] @Team great work adding the  MIT license, however, your link still doesn't work, you can see you created the MIT license file but when referencing it in the readme you use [MIT](./LICENSE), it would be nice if you ensured the link is working correctly.